### PR TITLE
adding note that cyberduck requires the full S3 path

### PR DESCRIPTION
**Pull request by mdodsworth**
@darwishoptly

### CODE REVIEW

**Comment by mdodsworth** [comment]
screen shot 2015-10-08 at 2 35 57 pm

File: src/content/rest/customer_profiles/customer_profiles/4-bulk_upload.md [hunk 1]
```diff
@@ -8,10 +8,14 @@ anchor: bulk
 You can also upload a CSV (comma-separated values) to the `optimizely-import` S3 bucket using the provided datasource S3
 path.  We will parse the given CSV, validate each row of data against the registered
 [attributes](/rest/customer_profiles#dcp_attributes), and store the successfully processed rows.  Each row is treated as
-a [write](/rest/customer_profiles#update-customer-profile) request.
+an [update](/rest/customer_profiles#update-customer-profile) request.
 
 Using the provided AWS credentials, it's possible to upload CSV files in a variety of ways.  The simplest approach is to
 use an S3 client application, such as [Cyberduck](http://www.cyberduck.io/?l=en).
+Note, Cyberduck requires the full S3 path, including both the `optimizely-import` bucket and given `s3_path`,
```
Comment (ajithopti) [inline src/content/rest/customer_profiles/customer_profiles/4-bulk_upload.md]: Perhaps `Note:` for consistency with other parts of the dev docs.

File: src/content/rest/customer_profiles/customer_profiles/4-bulk_upload.md [hunk 2]
```diff
@@ -8,10 +8,14 @@ anchor: bulk
...
+for example, `/optimizely-import`/`s3_path`.
```
Comment (ajithopti) [inline src/content/rest/customer_profiles/customer_profiles/4-bulk_upload.md]: You can remove the back-ticks around the slash before `s3_path`

**Comment by ajithopti** [comment]
LGTM. A couple of minor suggestions.

**Pull request merged**